models/services.ts

```diff
@@ -6,6 +6,8 @@ export type ResourceStatus =
   | 'Published'
   | 'Do Not Publish'
 
+export const NO_ASSOCIATED_GROUP = '-No Associated Group'
```

Comment:
Should check with the research team to see if we can just have resources w/o an associated group have that field as empty instead of this value